### PR TITLE
Translate Search Box

### PR DESCRIPTION
**Pull request by asantarissy**

### CODE REVIEW

File: _data/locales/en.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ en:
   locale_name: English
   subtitle: The missing package manager for macOS (or Linux)
   pagecontent:
+    search: Search
```
Comment (MikeMcQuaid) [inline _data/locales/en.yml]: can you add this to all `locales`? That'll allow removal of `if t.pagecontent.search` in `base.html`

**Comment by asantarissy** [comment]
@MikeMcQuaid done 🙂

File: _data/locales/en.yml (hunk 1 above)
Comment (asantarissy) [reply _data/locales/en.yml]: @MikeMcQuaid done 🙂

**Comment by asantarissy** [comment]
@MikeMcQuaid why did it fail? although it doesn't when I run it locally?

**Comment by MikeMcQuaid** [comment]
Rerunning CI, hopefully https://github.com/Homebrew/brew.sh/pull/341 fixed this.

**Comment by MikeMcQuaid** [comment]
Thanks again @asantarissy, great work!

**Pull request merged**